modules/integration/tests-ui/src/test/java/org/wso2/carbon/appmanager/integration/ui/Util/TestUtils/ApplicationInitializingUtil.java

```diff
@@ -100,9 +112,12 @@ public void testApplicationCreation(String prefix) throws Exception {
             policyPartialId = responseId.getString("id");
         }
 
+        String policyGropuId = appMPublisher.savePolicyGroup(xml, anonymousAccessToUrlPattern, policyGroupName,
+                throttlingTier, objPartialMappings, policyGroupDesc);
+
         int hostPort = 8080;
-        AppCreateRequest appCreateRequest = createSingleApp(appName+prefix, version, transport, appURL, hostPort,
-                appProp.getTier(), policyPartialId);
+        AppCreateRequest appCreateRequest = createSingleApp(appName+prefix, appDisplayName, version, transport, appURL, hostPort,
```

Comment:
This code segment has been refactored and this code line is not in here any more.